AppiumFuncTests/AppiumDriver/AppiumFuncTests.swift

```diff
@@ -193,4 +193,36 @@ class AppiumFuncTests: XCTestCase {
         guard let countAfter = eventsAfter?.events.count else { return }
         XCTAssertTrue(countAfter > countBefore)
     }
+
+    func testCanTypeTextOntoTextField() {
+        try! driver.findElement(by: .name, with: "TextFields").click()
+        let textFld = try! driver.findElement(by: .name, with: "Rounded")
+        textFld.click()
+        textFld.sendKeys(with: "Send Keys Test")
```

Comment:
Could you add unicode, too?
e.g. Emoji

Reply:
Sure, makes sense. I was also thinking that adding a non-latin alphabet
test l.e. Japanese or Russian would be a good. Any thoughts on that?